src/client.rs

```diff
@@ -473,16 +476,35 @@ pub enum ClientSet {
     ///
     /// Runs in `PostUpdate`.
     Send,
+    /// Systems that reset the client.
+    ///
+    /// Runs in `PreUpdate`.
+    ///
+    /// If this set is disabled, then you need to manually clean up the client after a disconnect or when
+    /// reconnecting.
+    Reset,
 }
 
 /// Last received tick for each entity.
 ///
 /// Used to avoid applying old updates.
+///
+/// If [`ClientSet::Reset`] is disabled, then this needs to be cleaned up manually.
 #[derive(Default, Deref, DerefMut, Resource)]
-pub(super) struct ServerEntityTicks(EntityHashMap<Entity, RepliconTick>);
+pub struct ServerEntityTicks(EntityHashMap<Entity, RepliconTick>);
```

Comment:
Do you need to view / remove / insert ticks here?
If not, could you export only `clear` as you did with `BufferedUpdates`?